typescript.js

```diff
@@ -114,6 +114,11 @@ module.exports = {
 						format: ['camelCase', 'snake_case', 'PascalCase', 'UPPER_CASE'],
 						leadingUnderscore: 'allow',
 					},
+					{
```

Comment:
can you add a test for this to make sure _html is now excluded form this rule?

Reply:
Added.